robottelo/ui/browser.py

```diff
@@ -1,6 +1,15 @@
+"""Tools to help getting a browser instance to run UI tests."""
+import time
+
 from robottelo.config import settings
 from selenium import webdriver
 
+try:
+    import docker
+except ImportError:
+    # Let fail later if not installed
```

Comment:
Exactly that way, this is an optional feature like previous virtual display. If not installed will raise an exception when trying to access the docker name which will be None. I don't want to introduce a new requirement just for a feature that will not be the default behavior and intended to be used most on a CI environment.